Service.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Task1_GUI
+{
+    internal class Service
+    {
+        public static string ErrorMessage { get; set; }
+
+        public static void AreAnyIdenticalCards(List<Card> cards1, List<Card> cards2)
```

Comment:
Based on the name of this method it should return some bool value. And I can't guess that it will throw an Exception